README.md

```diff
@@ -5,7 +5,7 @@ s2n-tls is a C99 implementation of the TLS/SSL protocols that is designed to be
 [![Build Status](https://codebuild.us-west-2.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiMndlTzJNbHVxWEo3Nm82alp4eGdGNm4rTWdxZDVYU2VTbitIR0ZLbHVtcFFGOW5majk5QnhqaUp3ZEkydG1ueWg0NGlhRE43a1ZnUzZaQTVnSm91TzFFPSIsIml2UGFyYW1ldGVyU3BlYyI6IlJLbW42NENlYXhJNy80QnYiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=main)](https://github.com/aws/s2n-tls/)
 [![Apache 2 License](https://img.shields.io/github/license/aws/s2n-tls.svg)](http://aws.amazon.com/apache-2-0/)
 [![C99](https://img.shields.io/badge/language-C99-blue.svg)](http://www.open-std.org/jtc1/sc22/wg14/www/docs/n1256.pdf)
-[![codecov](https://codecov.io/gh/aws/s2n-tls/branch/main/graph/badge.svg)](https://codecov.io/gh/aws/s2n-tls)
+[![codecov](https://codecov.io/gh/aws/s2n-tls/badge.svg?branch=main)](https://codecov.io/gh/aws/s2n-tls?branch=main)
```

Comment:
codecov.io doesn't play well with our safety macros: https://github.com/aws/s2n-tls/issues/3781. We should probably just remove it for now.

Reply:
Thanks, @lrstewart. What do you recommend, to close this  PR or  I can modify this to remove the codecov?

Reply:
I'd suggest modifying it to remove the codecov badge, if you're still interested in fixing the problem.

Reply:
I modified the PR. Thank you.